.github/workflows/pre-main.yaml

```diff
@@ -105,6 +105,7 @@ jobs:
         with:
           repository: test-network-function/cnf-certification-test-partner
           path: cnf-certification-test-partner
+          ref: 350f55ecddbb3b7804b85f444ae023edaf36adff
```

Comment:
Should not be needed anymore since  we now check for minikube in the partner repo